src/main/resources/com/google/cloud/tools/libraries/libraries.json

```diff
@@ -575,6 +575,30 @@
       }
     ]
   },
+{
```

Comment:
The indentation looks off?

Reply:
Your eyes are sharper than mine. Fixed. :-)